README.md

```diff
@@ -184,6 +184,30 @@ classroom-tool --unwatch
 This unsubscribes the logged-in user from all repositories in the configured
 classroom that match the configured basename.
 
+### Saving feedback on pull requests
+
+Passing `--save-feedback` will save a PDF of the "Files changed" tab for each
+student in the marking repository. Files will be stored in a `feedback/`
+subdirectory with the name of each file being `<github_username>.pdf`.
```

Comment:
I think this filename is now out if date, is that right?

Reply:
Oops. Fixed.